tsdb/shard_test.go

```diff
@@ -600,799 +600,874 @@ func TestShard_Close_RemoveIndex(t *testing.T) {
 
 // Ensure a shard can create iterators for its underlying data.
 func TestShard_CreateIterator_Ascending(t *testing.T) {
-	sh := NewShard()
+	var (
+		sh  *Shard
+		itr influxql.Iterator
+	)
```

Comment:
I personally loathe the block `var` declaration. :)

Why use four lines for what can be declared in two?